unitstatushandler/unitstatushandler.go

```diff
@@ -359,13 +359,13 @@ func (instance *Instance) CloudDisconnected() {
 func (instance *Instance) resetUnitStatus() {
 	instance.unitStatus = cloudprotocol.UnitStatus{
 		MessageType:  cloudprotocol.UnitStatusMessageType,
-		UnitConfig:   make([]cloudprotocol.UnitConfigStatus, 0),
-		Nodes:        make([]cloudprotocol.NodeInfo, 0),
-		Services:     make([]cloudprotocol.ServiceStatus, 0),
-		Instances:    make([]cloudprotocol.InstanceStatus, 0),
-		Layers:       make([]cloudprotocol.LayerStatus, 0),
-		Components:   make([]cloudprotocol.ComponentStatus, 0),
-		UnitSubjects: make([]string, 0),
+		UnitConfig:   nil,
```

Comment:
It means that we will omit these fields even for full unit status? Looks like it is not expected behavior

Reply:
As per public docs these fields have default value null.
I didn't find that null is applicable only if delta unit status is true.

So my assumption that should send json tags only if they have non-default values set (not null)